docs/source/design/composite-time-series.rst

```diff
@@ -0,0 +1,220 @@
++------------------------+------------------------------------------------------------------------------------------------------------------------+
+|Version                 |As Normal CWMS TS ID                                                                                                    |
++------------------------+------------------------------------------------------------------------------------------------------------------------+
+
+The zero's could also be var
+
+Composite Time Series Definition
+================================
+
+.. code-block::jsonc
+
+    {
+    "office": "<string>",   
+    "name": "<ts id name>",
+    "is-period-of-record": true, // or is authoritative. to distinguish between other possible use-cases?
+    "members": [
+        {
+            "time-series-id": "TS ID for this range",
+            "start": "start date of this",
+            "end": "end date of this range",
+            "notes": "text",
+            "version", "version date", // maybe not? could just use POR or period-of-record in the ts id version
+            // if values that equals the start or end timestamp are included
+            "start-inclusive": true,
+            "end-inclusive": false
+            // suggest default of "start-inclusive": true, "end-inclusive": false
+            // it may also make sense to just make this *always* the above and not let the user set it.
+            // alternatively if this is always [) then only start is required.
+            // however, the class would required and end field as the actual Time Series output needs to know the actual end.
+        }
+    ]
+    // array above *should* be sorted by start when provided to user.
+    }
+
+
+Operations required:
+
+* Create
+* Remove member (ts id + range)
+* Add member
+* List members
+* Replace all members?
+* Delete
+
+
+Composite Time Series Response
+==============================
+
+.. code-block::jsonc
+
+    {
+    // ... as current TimeSeries JSON
+    "composite-members-present": [
+        // member definition from above
+    ] 
+    }
+
+
+Supported Operations:
+
+* Get, through existing TimeSeries classes.
+
+Does it make sense to support writing directly to a composite time series. While the write of each element *could* be sent to the underlying member, this seems ripe for error when editing or updating any data. It is likely that any edits would always be to the most recent time series, and configured in some other system.
+
+Storage of member information
+================================
+
+#. Store in Clob as we refine the design
+#. Create appropriate tables once the design is stable.
+
+System responsibility for "knowing" to process composite.
+=========================================================
+
+Time Series Catalog
+-------------------
+
+Time Series Catalog should show composite time series and allow searching by "authoritative"
```

Comment:
just FYI: we'd have to teach CWMS-Vue a new Interval definition or a new Parameter Type in order to show up properly. I know you called up inevitable CWMS-Vue updates elsewhere, but wanted to be explicit.